drivers/pinctrl/pinctrl_ti_k3.c

```diff
@@ -27,10 +35,35 @@ int pinctrl_configure_pins(const pinctrl_soc_pin_t *pins, uint8_t pin_cnt, uintp
 	const struct device *dev = DEVICE_DT_GET(PINCTRL_NODE);
 	uintptr_t virt_reg_base = DEVICE_MMIO_GET(dev);
 
+	#ifdef CONFIG_SOC_AM62x_M4
```

Comment:
Does this unlocking need to be done on every call, or could it be done once during init?

Reply:
It should be done only once during init the way I understand it